pandas/core/style.py

```diff
@@ -771,3 +850,10 @@ def _highlight_extrema(data, color='yellow', max_=True):
                 extrema = data == data.min().min()
             return pd.DataFrame(np.where(extrema, attr, ''),
                                 index=data.index, columns=data.columns)
+
+
+def _maybe_wrap_formatter(formatter):
+    if not callable(formatter):
+        return lambda x: formatter.format(x)
+    else:
+        return formatter
```

Comment:
Good call. I now raise if it's not a callable or string.

This is probably something we should do more of. Since we compute the styles lazily, it'd be best to raise early when we can.